packages/@aws-cdk/aws-stepfunctions-tasks/lib/emrcontainers/create-virtual-cluster.ts

```diff
@@ -0,0 +1,154 @@
+import * as eks from '@aws-cdk/aws-eks';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
```

Comment:
It's more canonical to import the specific classes you need from core rather than the whole thing. Here I _think_ you can just get by with `import { Stack } from '@aws-cdk/core';`